examples/plot_employee_salaries.py

```diff
@@ -0,0 +1,106 @@
+"""
+Predicting the salary of employees
+==================================
+
+Now that we understand how the similarity encoder works, let's 
+see how this can affect prediction performance.
+
+The structure of this example will run as follows:
+* in the employee_salaries dataset we will first create a learning problem by 
+ chosing a column to predict
+* then we will benchmark the different kind of encodings (one-hot, similarity encoding, target encoding) 
+by fitting the target with their respective output with a Ridge Regression
+
+
+"""
+
+#########################################################################
+# 
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
You should do this import in the cell where it is used. It makes the example readable